bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -58,28 +55,35 @@ <h2 id="subtitle">{{ _('Own your life on every device with a Firefox Account.')
 </div>
 
 <div class="mzp-l-content">
-  {% call feature_card(
-    title=_('Send tabs instantly to your devices'),
-    image_url='firefox/accounts/fxa-send.png',
-    include_highres_image=True,
-    aspect_ratio='mzp-has-aspect-1-1',
-    class='mzp-l-card-feature-right-half mzp-t-firefox'
-  ) %}
-  <p>{{ _('Send an open tab on one device to all your others with a single tap. Much easier than texting or emailing yourself those links.') }}</p>
-  {% endcall %}
+
+  {% if l10n_has_tag('firefox_send_20190420') %}
+    {% call feature_card(
+      title=_('Take total control of the files you share'),
+      image_url='firefox/accounts/fxa-fxsend.png',
+      include_highres_image=True,
+      aspect_ratio='mzp-has-aspect-1-1',
+      class='mzp-l-card-feature-right-half mzp-t-firefox',
+      ga_title='Try Firefox Send'
+    ) %}
+    <p>{{ _('Feel like a secret agent when you send files online, with a private, encrypted link that ‘self-destructs.’') }}</p>
+    <a class="mzp-c-button mzp-t-secondary mzp-t-product" href="https://send.firefox.com/?utm_source=www.mozilla.org&amp;utm_medium=referral&amp;utm_campaign=fxa-features-nonfx&amp;utm_content=try-send&amp;entrypoint=mozilla.org-fxa.features" data-link-name="Try Firefox Send" data-link-type="button">{{ _('Try Firefox Send') }}</a>
+    {% endcall %}
+  {% endif %}
 
   {% call feature_card(
     title=_('Remember your passwords in Lockbox'),
     image_url='firefox/accounts/fxa-lockbox.png',
     include_highres_image=True,
     aspect_ratio='mzp-has-aspect-1-1',
-    class='mzp-l-card-feature-left-half mzp-t-firefox lockbox-card',
-    link_cta=_('Get the Lockbox App'),
-    link_url='https://app.adjust.com/kq8zhhz',
+    class='mzp-l-card-feature-left-half mzp-t-firefox lockbox-card qr-modal',
     ga_title='Get the Lockbox App'
   ) %}
     <p>{{ _('No more getting locked out of apps and websites. Lockbox secures all the passwords you’ve saved in Firefox and gives you easy access across all your iOS devices.') }}</p>
```

Comment:
This seems like a good change, but the plumbing doesn't seem to be in place for it. The QR code talks about Apple. I just don't feel confident that we have all the bits in place to accomplish it in this PR, and I don't want to delay this PR while we figure them out. A follow-up issue seems reasonable!

Reply:
Well my only misgivings there are that it will mean even more string changes in the very near future, I think it would be better on l10n to bundle them up. Maybe we can resolve this quickly and not delay the PR. Who do we need to talk to?

Reply:
The `https://app.adjust.com/kq8zhhz` has already been updated to be cross-platform, so I think all the plumbing is in order, we just need to update copy.

Reply:
So let's change this to "across all your Android and iOS devices" (behind the same l10n tag)

Reply:
Done.